bitcoin-magazine/bitcoin-explained/op_return.md

```diff
@@ -0,0 +1,596 @@
+---
+title: "OP_RETURN"
+transcript_by: realdezzy via review.btctranscripts.com
+media: https://www.youtube.com/watch?v=NYj80OGlWGg
+tags: ["script","utxo"]
+speakers: ["Sjors Provoost","Aaron Van Wirdum"]
+categories: ["Podcast"]
+date: 2022-07-15
+---
+
+## Intro
+
+Aaron Van Wirdum: 00:00:19
+
+Live from Utrecht, this is Bitcoin Explained.
+Hey Sjors.
+
+Sjors Provoost: 00:00:24
+
+What's up?
+
+Aaron Van Wirdum: 00:00:25
+
+How exciting.
+Two weeks ago you told me that maybe you would go to Bitcoin Amsterdam.
+
+Sjors Provoost: 00:00:31
+
+Yes.
+
+Aaron Van Wirdum: 00:00:31
+
+And now you're a speaker.
+
+Sjors Provoost: 00:00:33
+
+I'm a panelist, probably not a real speaker.
+
+Aaron Van Wirdum: 00:00:37
+
+That counts that that's a speaker in my book Sjors.
+
+Sjors Provoost: 00:00:40
+
+Sounds good.
+
+Aaron Van Wirdum: 00:00:41
+
+How exciting.
+Sjors today we're gonna discuss we're gonna discuss a blog post by BitMax Research.
+
+Sjors Provoost: 00:00:48
+
+That's right.
+
+Aaron Van Wirdum: 00:00:48
+
+Which is about the OP_RETURN Wars.
+So exactly, very nice clickbait title.
+Do you remember what the name of the blog post is?
+Was it was it just the OP_RETURN Wars?
+
+## OP_RETURN War
+
+Sjors Provoost: 00:01:00
+
+It's called the OP_RETURN wars of 2014, Dapps versus Bitcoin transactions.
+And it's a slightly clickbaity title because well it was only called wars by certain other people.
+But
+
+Aaron Van Wirdum: 00:01:14
+
+wasn't it referred to as the OP_RETURN wars previously in your memory?
+
+Sjors Provoost: 00:01:18
+
+Yeah, but some people object that the OP_RETURN wars have been sort of that term was phrased by people trying to launch various coins and trying to make Bitcoin developers look bad.
+Right.
```

Comment:
There are instances where phrases like 'Right' or 'Yeah' from one speaker mistakenly appear in the text attributed to another speaker. Please remove these occurrences for clarity and accuracy.